embedded-graphics/README.tpl

```diff
@@ -0,0 +1,82 @@
+<p align="center">
+    <img width="150" src="https://raw.githubusercontent.com/jamwaffles/embedded-graphics/191fe7f8a0fedc713f9722b9dc59208dacadee7e/assets/logo.svg?sanitize=true" alt="Embedded graphics logo">
+</p>
+<p align="center">
+    <a href="https://circleci.com/gh/jamwaffles/embedded-graphics/tree/master"><img src="https://circleci.com/gh/jamwaffles/embedded-graphics/tree/master.svg?style=shield" alt="Build Status"></a>
+    <a href="https://crates.io/crates/embedded-graphics"><img src="https://img.shields.io/crates/v/embedded-graphics.svg" alt="Crates.io"></a>
+    <a href="https://docs.rs/embedded-graphics"><img src="https://docs.rs/embedded-graphics/badge.svg" alt="Docs.rs"></a>
+    <a href="https://matrix.to/#/#rust-embedded-graphics:matrix.org"><img src="https://img.shields.io/matrix/rust-embedded-graphics:matrix.org" alt="embedded-graphics on Matrix"></a>
+</p>
+
+# Embedded graphics
+
+{{readme}}
+
+## Generating readmes
```

Comment:
Do we need to include this in the user facing readme file? I would suggest to move everything that isn't directly related to using the library to a new file in the `doc` directory and only add a link to that file to the readme.

Reply:
Good point. I've moved it into `doc/`, along with the development setup section.